app/controllers/manage_assessments/coverages_controller.rb

```diff
@@ -2,7 +2,12 @@ module ManageAssessments
   class CoveragesController < ApplicationController
     def new
       @coverage = course.coverages.build
-      @coverage.outcome_coverages.build
+      if outcome_id_given_in_query_params
```

Comment:
Do we need the conditional? What happens if we *always* do:

```
@coverage.outcome_coverages.build(outcome_id: params[:outcome_id])
```

If the param is `nil` it's no different than `build` with no params. Is there a problem if I pass an invalid id?